### PR TITLE
MC-10288: Adding docs for create pricing

### PR DESCRIPTION
**Pull request by evanluc**
### Fixes [MC-10288](https://cloud-ops.atlassian.net/browse/MC-10288)

Adding documentation for create pricing. I did my best to abide by our new template. Hopefully I didn't miss anything @patrickspensieri 😁

**_Wait until list merge to merge this_**

### CODE REVIEW

File: source/includes/administration/_pricings.md [hunk 1]
```diff
@@ -231,3 +231,133 @@ Attributes | &nbsp;
...
+  }
+}
+```
+
+Required | &nbsp;
+------- | -----------
+`organization.id` <br/>*UUID* | The id of the organization the pricing belongs to. 
+`productCatalogs` <br/>*Array[productCatalogs.id]* | An array of product catalogs the pricing can add products from.
+`productCatalogs.id` <br/>*UUID* | The id for a product catalog you'd like to add as a source for your pricing.
+`name` <br/>*Map[string,string]* | A map of language keys to a name in the specified language.
```
Comment (fyrnaga) [inline source/includes/administration/_pricings.md]: I know that this is a map in our structure. But is in in json?

Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: We should have the same definition than the list.  So we should correct one or the other.

File: source/includes/administration/_pricings.md [hunk 2]
```diff
@@ -231,3 +231,133 @@ Attributes | &nbsp;
...
+}
+```
+
+Required | &nbsp;
+------- | -----------
+`organization.id` <br/>*UUID* | The id of the organization the pricing belongs to. 
+`productCatalogs` <br/>*Array[productCatalogs.id]* | An array of product catalogs the pricing can add products from.
+`productCatalogs.id` <br/>*UUID* | The id for a product catalog you'd like to add as a source for your pricing.
+`name` <br/>*Map[string,string]* | A map of language keys to a name in the specified language. 
+`description` <br/>*Map[string,string]* | A map of language language keys to the description in the specified language.
```
Comment (fyrnaga) [inline source/includes/administration/_pricings.md]: same as for name

File: source/includes/administration/_pricings.md [hunk 3]
```diff
@@ -231,3 +231,133 @@ Attributes | &nbsp;
...
+
+Create a new pricing.
+
+```shell
+# Creates a new pricing
+curl -X POST "https://cloudmc_endpoint/rest/pricings" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> Request body example(s):
```
Comment (patrickspensieri) [inline source/includes/administration/_pricings.md]: We can make this singular, I'd added the **(s)** for when we've got more than one

File: source/includes/administration/_pricings.md [hunk 4]
```diff
@@ -231,3 +231,133 @@ Attributes | &nbsp;
...
+			"currency": "USD"
+		}
+	],
+	"supportedCurrencies": [
+		"CAD",
+		"USD"
+	]
+}
+```
+> The above command(s) return(s) JSON structured like this:
```
Comment (patrickspensieri) [inline source/includes/administration/_pricings.md]: Same here! If it's unclear in the docs template then I could remove the **(s)** from both

**Review by patrickspensieri — APPROVED** [review]
Looks good!

File: source/includes/administration/_pricings.md (hunk 3 above)
Comment (evanluc) [inline source/includes/administration/_pricings.md]:
```suggestion
> Request body example:
```

File: source/includes/administration/_pricings.md (hunk 4 above)
Comment (evanluc) [inline source/includes/administration/_pricings.md]:
```suggestion
> The above command return JSON structured like this:
```

Comment (evanluc) [inline source/includes/administration/_pricings.md:295]:
```suggestion
> The above command(s) return JSON structured like this:
```

Comment (evanluc) [reply source/includes/administration/_pricings.md]: https://github.com/cloudops/cloudmc-api-docs/pull/142/commits/d9573b438f8d2acd1e143738ca1c30e46b03d89f

File: source/includes/administration/_pricings.md (hunk 3 above)
Comment (evanluc) [reply source/includes/administration/_pricings.md]: https://github.com/cloudops/cloudmc-api-docs/pull/142/commits/d9573b438f8d2acd1e143738ca1c30e46b03d89f

File: source/includes/administration/_pricings.md (hunk 1 above)
Comment (evanluc) [reply source/includes/administration/_pricings.md]: I'll change it object

File: source/includes/administration/_pricings.md [hunk 5]
```diff
@@ -231,3 +231,132 @@ Attributes | &nbsp;
...
+  }
+}
+```
+
+Required | &nbsp;
+------- | -----------
+`organization.id` <br/>*UUID* | The id of the organization the pricing belongs to. 
+`productCatalogs` <br/>*Array[productCatalogs.id]* | An array of product catalogs the pricing can add products from.
+`productCatalogs.id` <br/>*UUID* | The id for a product catalog you'd like to add as a source for your pricing.
+`name` <br/>*Map[string,string]* | A map of language keys to a name in the specified language.
```
Comment (evanluc) [inline source/includes/administration/_pricings.md]:
```suggestion
`name` <br/>*Object* | A map of language keys to a name in the specified language. 
```

File: source/includes/administration/_pricings.md [hunk 6]
```diff
@@ -231,3 +231,132 @@ Attributes | &nbsp;
...
+}
+```
+
+Required | &nbsp;
+------- | -----------
+`organization.id` <br/>*UUID* | The id of the organization the pricing belongs to. 
+`productCatalogs` <br/>*Array[productCatalogs.id]* | An array of product catalogs the pricing can add products from.
+`productCatalogs.id` <br/>*UUID* | The id for a product catalog you'd like to add as a source for your pricing.
+`name` <br/>*Map[string,string]* | A map of language keys to a name in the specified language. 
+`description` <br/>*Map[string,string]* | A map of language language keys to the description in the specified language.
```
Comment (evanluc) [inline source/includes/administration/_pricings.md]:
```suggestion
`description` <br/>*Object* | A map of language language keys to the description in the specified language.
```

File: source/includes/administration/_pricings.md (hunk 2 above)
Comment (evanluc) [reply source/includes/administration/_pricings.md]: https://github.com/cloudops/cloudmc-api-docs/pull/142/commits/5a17a84d28c2a86b44de607482ee752c52102de4

File: source/includes/administration/_pricings.md (hunk 1 above)
Comment (evanluc) [reply source/includes/administration/_pricings.md]: https://github.com/cloudops/cloudmc-api-docs/pull/142/commits/5a17a84d28c2a86b44de607482ee752c52102de4

**Review by fyrnaga — APPROVED** [review]
LGTM

**Pull request merged**